db/migrate/20190924173755_delete_policies.rb

```diff
@@ -1,5 +1,14 @@
 class DeletePolicies < ActiveRecord::Migration[5.2]
-  def change
+  def up
     drop_table :policies
```

Comment:
This is not an automatically reversible migration and so requires up/down rather than change if we want to support rollback.